requirements/test.txt

```diff
@@ -1,5 +1,5 @@
 -r base.txt
-pytest==3.5.0
+pytest>3.5.0
```

Comment:
Should we just pin this to the appropriate version or completely drop the version pinning for pytest?

Reply:
I think dropping version pinning is better. Pinning is what caused a problem here. I would have however done `>=`. But that might not upgrade the package installed because the current 3.5.0 will anyways satisfy.